### PR TITLE
OSS CI Updates

### PR DESCRIPTION
**Pull request by prateekdesai04**
***Description of changes:***
Making changes to CI, removing pre-cloned repository pre-OSS.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: graphstorm_job.sh [hunk 1]
```diff
@@ -1,30 +1,38 @@
...
+git clone https://github.com/awslabs/graphstorm.git
+
+cd graphstorm
+
+if [ ! -z $REMOTE ]; then
+    git remote set-url origin $REMOTE
+fi
+
+git fetch origin $SOURCE_REF:working
+git checkout working
```
Comment (classicsong) [inline graphstorm_job.sh:26]: Why checkout working?

**Review by classicsong — APPROVED** [review]

File: graphstorm_job.sh (hunk 1 above)
Comment (prateekdesai04) [reply graphstorm_job.sh:26]: We create a new branch called working, fetch the PR code in that branch and git checkout working switches the current working branch to 'working', which is now pointing to the same commit as the source branch. This allows us to perform subsequent Git operations, such as building or testing the code, on the source branch.

Comment (classicsong) [reply graphstorm_job.sh:29]: But what if the branch name is not working?

Comment (prateekdesai04) [reply graphstorm_job.sh:29]: working is a branch which we create in the github runners after cloning the repository and we checkout code on this 'working' branch and execute tests in this branch

**Pull request merged**